content/en/contribute/code/workflow.md

```diff
@@ -14,6 +14,17 @@ Where possible, follow our [coding style guide]({{< ref "contribute/code/style-g
 
 Aim for self-documenting code. Where code cannot be made self-documenting add commenting. Usually comments are useful when they explain why some code exists, and should not be explaining what some code is doing. 
 
+### Pushing Code & Opening Pull Requests
+
+While working it is recommended that you push to your branch often. Others can see your progress, but will be aware it is a work-in-progress.
+
+To collaborate with others it is recommended that you use GitHub tooling, namely Pull Requests. When opening a pull request, consider the following:
```

Comment:
I don't think these sentences are needed. Line 19 is covered better by the workflow on line 26, and line 21 is explained better in the list items below.